.circleci/config.yml

```diff
@@ -1,110 +1,81 @@
 version: 2.1
+orbs:
+    browser-tools: circleci/browser-tools@1.2.4
+    node: circleci/node@5.0.2
+    commitlint: conventional-changelog/commitlint@1.0.0
 aliases:
   - &defaults
       docker:
-        - image: cimg/node:12.22.11-browsers
+        - image: cimg/node:lts-browsers
           auth:
             username: $DOCKERHUB_USERNAME
             password: $DOCKERHUB_PASSWORD
       working_directory: ~/repo
       environment:
         NODE_ENV: production
   - &setup
-    name: "setup"
-    command: |
-      npm --production=false ci
-      mkdir ./test/results
+    node/install-packages:
+      override-ci-command: npm --production=false ci
```

Comment:
Sorry for all the visual noise here. The changes in the `aliases` section are:

1. Convert all of these from just `name, command` to `run: {name: command}` so that step 2 is possible
2. Replace the contents of `setup` with the `node/install-packages` step from the `node` orb, instead of the previous `run` step.

Reply:
Nice! Thanks for the explanation here.